### PR TITLE
feat(staking): add retired and saturated indicators

### PR DESCRIPTION
**Pull request by szymonmaslowski**
# Checklist

- [x] JIRA - [\<link>](https://input-output.atlassian.net/browse/LW-7545)
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Showing red moon icon when pool is retired
Showing red warning icon when pool has >100% saturation

## Testing

IDK TBH... If there is a way to stake on pools that will get over saturated and retired then we could test that...

## Screenshots

image
image

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/1044/5588834579/index.html) for [42193aec](https://github.com/input-output-hk/lace/pull/286/commits/42193aec6d61aa7ee780af30cca4c4252e4abd19)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 35     | 0      | 0       | 0     | 35    | ✅     |

### CODE REVIEW

File: packages/staking/src/features/overview/staking-info-card/StakingInfoCard.tsx [hunk 1]
```diff
@@ -70,15 +90,20 @@ export const StakingInfoCard = ({
...
               logo={logo ?? getRandomIcon({ id: id.toString(), size: 30 })}
               name={name}
               ticker={ticker}
               id={id}
               onClick={onStakePoolSelect}
             />
-          </div>
+            {(status === 'retired' || status === 'saturated') && (
+              <Tooltip content={t(mapOfStatusToLabel[status])}>
+                <Icon style={{ color: '#FF5470', fontSize: '24px' }} component={mapOfStatusToIcon[status]} />
```
Comment (mrcnk) [inline packages/staking/src/features/overview/staking-info-card/StakingInfoCard.tsx:103]: We can probably export this color to variable (seen there is inline color in StakingConfirmation).

File: packages/staking/src/features/overview/staking-info-card/StatsTooltip.tsx [hunk 2]
```diff
@@ -5,23 +5,25 @@ import styles from './Stats.module.scss';
...
   title?: string | number;
   children: string | React.ReactElement | React.ReactNode;
+  content?: React.ReactNode;
 }): React.ReactElement => {
-  if (!title) return <>{children}</>;
+  const body =
+    content ||
+    (title && (
+      <>
+        <div>{'USD Value'}</div>
```
Comment (mrcnk) [inline packages/staking/src/features/overview/staking-info-card/StatsTooltip.tsx:18]: Would be awesome to have i18n 😄

**Review by mrcnk — APPROVED** [review]

File: packages/staking/src/features/overview/staking-info-card/StakingInfoCard.tsx (hunk 1 above)
Comment (szymonmaslowski) [reply packages/staking/src/features/overview/staking-info-card/StakingInfoCard.tsx:103]: I think we could fix that during refactoring. This color is defined in the toolkit so we will access this color from theme.

File: packages/staking/src/features/overview/staking-info-card/StatsTooltip.tsx (hunk 2 above)
Comment (szymonmaslowski) [reply packages/staking/src/features/overview/staking-info-card/StatsTooltip.tsx:18]: I will add it in the subsequent PR 😉

**Pull request merged**